src/fileServices/s3Service.ts

```diff
@@ -41,7 +42,7 @@ export class S3Service implements FileService {
     const { s3Bucket: bucket } = this.options;
 
     const params = {
-      CopySource: originalPath,
+      CopySource: `${bucket}/${originalPath}`,
       Bucket: bucket,
```

Comment:
I guess it was needed, but I don't get why we need the bucket parameter AND the bucket in the "key".

Reply:
I think it's because you can copy for an other bucket